manifest/xsd/tract.xsd

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
These are the actual schemas for validating tract xml. If we generate something that doesn't follow our spec these schemas will be able to report what is invalid about the XML. I want these to be utilized in the API when creating/managing pages so we can guarantee that we will be working against expected XML and not introduce bugs that end up becoming part of the spec.